recipes/dj-static/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set package = "dj-static" %}
```

Comment:
TBH, I'm not a fan of including any variables at the top. It takes me an extra step to build them using `conda build .` locally. I was just going in the direction of using variables at the top of the file. I really don't see a difference between using 1 or 3... but there is considerable difference between using 0 and 1.